playbooks/roles/kibana/templates/nginx.conf.j2

```diff
@@ -0,0 +1,50 @@
+upstream elasticsearch_server {
+   server 127.0.0.1:9200;
+}
+
+server {
+  # Kibana server, templated by ansible
+
+  {% if NGINX_ENABLE_SSL %}
+
+  listen {{ KIBANA_SERVER_NAME }}:{{ KIBANA_NGINX_PORT }};
+  listen {{ KIBANA_SERVER_NAME }}:{{ KIBANA_SSL_NGINX_PORT }};
+
+  ssl_certificate /etc/ssl/certs/{{ NGINX_SSL_CERTIFICATE|basename }};
```

Comment:
This is the standard way edx's nginx handles certs. NGINX_SSL_CERTIFICATE is in nginx/defaults/main, but the default value does not exist.